release-notes.md

```diff
@@ -1,9 +1,86 @@
 ---
-layout: archive
-title: "Release Notes"
+layout: post
+title: Armory Release Notes
 permalink: /release-notes/
-description: Add a description here.
+description: Release Notes For Armory Spinnaker
 collection: release_notes
 ---
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
 
-This is a placeholder to point at the custom archive.html layout.
+
+## Understanding Armory + Open Source Spinnaker releases
+Armory is based off of OSS's (Open Source Spinnaker) [Halyard release Cadence](https://www.spinnaker.io/community/releases/release-cadence), in which we extend OSS with Armory features. We provide a few different release types.
+
+A **stable** release for customers who need stability in their production environment. You can find these [releases here](#stable-armory-releases) or use [Armory Halyard](#differences-between-releases).
+
+For customers who want to test the latest Armory + OSS (Open Source Spinnaker) release, we provide **rc** release candidates, which you can find using [Armory Halyard](#differences-between-releases). See OSS's [Release Cadence](https://www.spinnaker.io/community/releases/release-cadence/) for more information.
+
+We also provide nightly **ossedge** builds from **OSS master**, which is only built, but not tested by Armory, which you can find using [Armory Halyard](#differences-between-releases).
+
+
+### Differences between Releases
+Given this example:
+- ...
+- `OSS 1.9.1 + Armory`
+- `OSS 1.9.2 + Armory <-- "Armory stable"`
+- `OSS 1.9.2+more + Armory <-- "Armory rc"` (**more** means commits created in OSS for `OSS 1.9.3`, but not officially released)
+- `OSS 1.10.0 + Armory`
+- `OSS 1.10.1 + Armory <-- "Armory next rc"`
+- `OSS master <-- "ossedge"`
+
+
+#### Stable Releases
+```bash
+$ hal version list
+```
+A **stable** release is created using the OSS `v1.9.2`
+- This version has been tested at Armory and verified that it will work for our normal uses cases.
+- Most of our customers will be using stable releases.
+
+We use [semantic versioning](https://semver.org/) for tagging, e.g. `12.3.4`.
```

Comment:
Looks like we're a little inconsistent in referring to versions as either "v1.9.2" or "1.9.2".  We should determine whether we want to refer to versions with a v in front or not, and then make sure all our release version references here match.